skiplang/prelude/src/skstore/EagerFilter.sk

```diff
@@ -93,12 +91,12 @@ class EagerFilter private {
   private static fun mkDir(
     context: mutable Context,
     childName: DirName,
-    suffix: String,
+    suffix: String = "",
```

Comment:
Would it make sense to use `suffix: ?String` here?

Reply:
Why not. `mkDir` is only ever used twice, privately, on lines 249, 250.

Reply:
Done